src/webview/leetCodePreviewProvider.ts

```diff
@@ -14,22 +16,29 @@ class LeetCodePreviewProvider implements Disposable {
         this.context = context;
     }
 
-    public async preview(node: IProblem): Promise<void> {
+    public async show(node: IProblem): Promise<void> {
+        // Fetch problem first before creating webview panel
+        const descString: string = await leetCodeExecutor.getDescription(node);
+
         this.node = node;
         if (!this.panel) {
-            this.panel = window.createWebviewPanel("leetcode.preview", "Preview Problem", ViewColumn.Active, {
+            this.panel = window.createWebviewPanel("leetcode.preview", "Preview Problem", ViewColumn.One, {
```

Comment:
`ViewColumn.One` or `ViewColumn.Two`?

Reply:
The panel is created to Column 1, but moved to Column 2 after creating code file.